bin/setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+echo ">> Install python requirements from 'requirements-dev.txt'"
+pip install -r requirements-dev.txt > /dev/null
```

Comment:
This will still display the errors, it just redirects the stdout, while displaying the stderr to console.